docs/UserGuide.adoc

```diff
@@ -232,7 +238,7 @@ Format: `undo`
 
 [NOTE]
 ====
-Some commands are undoable. e.g. `wipe PlanMySem`.
+Some commands are undoable. e.g. `clear`.
```

Comment:
I think we should specify all commands that are undoable or the user might not know which other commands are undoable.

Reply:
Actually, only the clear command is undoable so far. I'll make that clear.